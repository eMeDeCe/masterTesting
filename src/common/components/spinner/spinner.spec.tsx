import * as React from 'react';
import { render } from '@testing-library/react';
import { SpinnerComponent } from './spinner.component';

describe('SayHello component specs', () => {
  it('TEST 1 SPINNER: render correctly Spinner component', () => {
    // Arrange
    // Act
    const { asFragment } = render(<SpinnerComponent />);

    // Assert
    expect(asFragment()).toMatchSnapshot();
  });
});
