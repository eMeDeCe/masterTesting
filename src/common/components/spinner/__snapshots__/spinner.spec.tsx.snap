// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SayHello component specs TEST 1 SPINNER: render correctly Spinner component 1`] = `<DocumentFragment />`;
